src/Lecture1.hs

```diff
@@ -119,4 +147,22 @@ and lower than 6 elements (4, 5, 6, 7, 8 and 9).
 
 🕯 HINT: Use recursion to implement this function.
 -}
-lowerAndGreater n list = error "TODO"
+
+getMinMax :: Int -> Int -> Int -> [Int] -> [Int]
+getMinMax min max n list = go min max list
```

Comment:
I see that GHC outputs some warnings ⚠️
In this case, it happens because `max` and `min` are the names of standard functions in Prelude. It still works due to name shadowing. But generally, it's better to avoid name conflicts with standard functions.

It's a bit tough to do this here because both `max` and `maximum` are already taken 😅 But I would use smth like `mx` or `maxVal` (and similar for minimum).